Rakefile

```diff
@@ -1,22 +1,18 @@
 # frozen_string_literal: true
 
 require "bundler/gem_tasks"
-require "rake/testtask"
 Dir["tasks/**/*.rake"].each { |t| load t }
 
-Rake.application.options.trace = false
+require "rubocop/rake_task"
+RuboCop::RakeTask.new
 
-Rake::TestTask.new do |t|
-  t.libs << "lib"
-  t.libs << "spec"
-  t.warning = false
-  t.test_files = FileList["spec/**/*_spec.rb"]
+desc "Run tests"
+task :test do
+  require "shellwords"
+  test = Array(ENV.fetch("TEST", []))
```

Comment:
Is it possible to pass more than one test via `TEST`? Should this be a string instead of an array?

Reply:
TIL